packages/file_selector/file_selector_android/android/src/main/java/io/flutter/plugins/file_selector/PathUtils.java

```diff
@@ -0,0 +1,112 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.file_selector;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.OpenableColumns;
+import androidx.annotation.NonNull;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.util.ArrayList;
+
+public class PathUtils {
+
+  static String cacheFolder = "file_selector";
```

Comment:
Is this assumed to be the same cache as the one defined in [FileSelectorDelegate.java](https://github.com/flutter/plugins/pull/6547/files#diff-233f4faa39ffe628203060d7feae95b567bf35f46d0685fd91dc550e3bc52c56) if so then consider using a shared static variable.